internal/api/design/package_.go

```diff
@@ -76,16 +76,16 @@ var _ = Service("package", func() {
 			})
 			AttributeUUID("location_id", "Identifier of storage location")
 			Attribute("status", String, func() {
-				EnumPackageStatus()
+				EnumSIPStatus()
 			})
 			Attribute("limit", Int, "Limit number of results to return")
 			Attribute("offset", Int, "Offset from the beginning of the found set")
 
 			Token("token", String)
 		})
-		Result(PackageList)
+		Result(SIPs)
 		HTTP(func() {
-			GET("/")
+			GET("/sip")
```

Comment:
I think using plural nouns for collection identifiers is considered best practice, as suggested in these articles:

- https://swagger.io/resources/articles/best-practices-in-api-design/
- https://learn.microsoft.com/en-us/azure/architecture/best-practices/api-design#organize-the-api-design-around-resources
- https://google.aip.dev/122#collection-identifiers

Reply:
A good moment to change that, thanks!

Reply:
I've also used them in the ABAC scopes.